ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -218,17 +221,19 @@ export const AdminUserTable = withUserProfile()(
         firstSignInTimestamp: user.firstSignInTime,
         institutionName: this.displayInstitutionName(user),
         name: (
-          <StyledRouterLink
-            path={`/admin/users/${usernameWithoutDomain(user.username)}`}
-            target='_blank'
-          >
+          <AdminUserLink username={user.username} target='_blank'>
```

Comment:
Do we want this to open in a new tab? It looks like this was the behavior from before, but I was curious if we might want to change this.

Reply:
We were talking in mob about whether or not we want to include the domain for the username on this link. It seems as though everyone would have the same domain, so we might be able to save some space by dropping the domain. What do you think?

Reply:
remove domain is https://precisionmedicineinitiative.atlassian.net/browse/RW-8256

Reply:
(this PR now removes the domain, but retains new-tab behavior)

Reply:
The admins have confirmed that they prefer new-tab, so no changes are needed.

Reply:
Thanks for checking.